models/public/rexnet-v1-1.0x/rexnet-v1-1.0x.md

```diff
@@ -0,0 +1,92 @@
+# rexnet-v1-1.0x
+
+## Use Case and High-Level Description
+
+ReXNet V1 1.0x is image classification model pretrained on ImageNet dataset. Rank eXpansion Network(ReXNet): Diminishing Representational Bottleneck on Convolutional Neural Network.
```

Comment:
this text is not logically connected, of I right understand, `Rank eXpansion Network(ReXNet): Diminishing Representational Bottleneck on Convolutional Neural Network.` it is name of paper.

Reply:
Updated, please take a look